spaceship/lib/spaceship/tunes/build_train.rb

```diff
@@ -97,8 +97,9 @@ def setup
           # |       build.valid = | false             | true              | false           | true               | true    |
           # | .processing_state = | "processing"      | "processing"      | "invalidBinary" | "processingFailed" | nil     |
           # +---------------------+-------------------+-------------------+-----------------+--------------------+---------+
-          @processing_builds << build if build.processing_state == 'processing' ||
-            (build.processing && build.processing_state != 'invalidBinary' && build.processing_state != 'processingFailed')
+          if build.processing_state == 'processing' || (build.processing && build.processing_state != 'invalidBinary' && build.processing_state != 'processingFailed')
```

Comment:
@KrauseFx As I note in the comment above, everything after the `||` is included just because the pre-existing test cases will otherwise fail. Based on my admittedly limited testing, though, just checking that `processing_state == 'processing` should be enough. I'd want other people to confirm my findings before removing/editing those old test cases and simplifying this conditional.